miniconda3-nanoserver/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/windows/servercore:1809 AS servercorebase
```

Comment:
Any thoughts on using nanoserver here? I think it would be neat because it's so much more lean but when I tried it, it looks like it didn't work because it does not have powershell